### DISCUSSION TITLE
Trubble with numpy np.int

### DISCUSSION THREAD

**Discussion by Sebastianostby** [shap/shap#2982] [Q&A] [answered]
Hi, 
whenever i try to run shap explainers i get the following error code:
""
             /envs/shapTest/lib/python3.9/site-packages/shap/explainers/_kernel.py", line 277, in explain
num_subset_sizes = np.int(np.ceil((self.M - 1) / 2.0))

AttributeError: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
""
Have not been able to solve this, tried different evns and a older version of numpy.

Any suggestions that is not brute-force changing the source code of the package?

**Comment by Sebastianostby** [comment, marked as answer]
Upgrading the pip installation fixed the problem!

**Reply by bmoallemi** [reply]
I'm still getting this error, can you say more?

**Reply by bmoallemi** [reply]
Nevermind, just upgrading the pip installation indeed did the trick!
